src/queries/auth/index.tsx

```
@@ -0,0 +1,67 @@
+  setErrorMessage: (message: string) => void
+) => {
+  const router = useRouter()
+  const login = useAuthStore(state => state.login)
+
+  return useMutation({
+    mutationFn: SignIn,
+    onSuccess: data => {
+      login(data.result)
+      router.push('/')
+    },
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)